<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTableForm</class>
 <widget class="QWidget" name="MyTableForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор товаров</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="labeltable">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="buttonss">
     <property name="spacing">
      <number>14</number>
     </property>
     <property name="leftMargin">
      <number>150</number>
     </property>
     <property name="rightMargin">
      <number>150</number>
     </property>
     <item>
      <widget class="QPushButton" name="closeButton2">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите категорию товара для поиска</string>
   </property>
  </widget>
  <zorder>labeltable</zorder>
  <zorder>tableView</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>searchLineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
